include/WCSimPrimaryGeneratorAction.hh

```diff
@@ -251,6 +260,14 @@ private:
   inline void SetIBDModel(G4String choice) { ibd_model = choice; }
   inline G4String GetIBDModel()  { return ibd_model; }
 
+  // HEPMC3 reader
+  inline void SetHepMC3EvtGenerator(G4bool choice) { useHepMC3Evt = choice; }
+  inline G4bool IsUsingHepMC3EvtGenerator()  { return useHepMC3Evt; }
+  inline void SetHepMC3Filename(G4String choice) { hepmc3_filename = choice; }
+  inline G4String GetHepMC3Filename()  { return hepmc3_filename; }
+  inline void SetHepMC3PositionGen(G4bool choice) { hepmc3_positionGen = choice; }
+  inline G4bool SetHepMC3DirectionGen(G4bool choice) { return hepmc3_positionGen;}
```

Comment:
```suggestion
```
I *think* this isn't used

Alternatively, keep it but fix it! It currently sets `hepmc3_positionGen` (i.e. position, not direction)

Reply:
I must have been in a rush on the day I wrote this. It's meant to be `GetHepMC3PositionGen` and return the bool. Solved this now!